wdfn-server/waterdata/schema.py

```diff
@@ -14,50 +12,39 @@
 from .utils import parse_rdb
 
 
-SERVICE_ROOT = app.config['SERVER_SERVICE_ROOT']
+SERVICE_ROOT = app.config["SERVER_SERVICE_ROOT"]
```

Comment:
Just a nit: We use single quotes in the python code rather than double quotes.

Reply:
Oh okay.  I will make that change in a separate PR.